lint/linter.py

```diff
@@ -15,6 +16,18 @@
 ARG_RE = re.compile(r'(?P<prefix>@|--?)?(?P<name>[@\w][\w\-]*)(?:(?P<joiner>[=:])(?:(?P<sep>.)(?P<multiple>\+)?)?)?')
 BASE_CLASSES = ('PythonLinter',)
 
+MATCH_DICT = OrderedDict({
+    "match": None,
+    "line": None,
+    "col": None,
+    "error": None,
+    "warning": None,
+    "message": '',
+    "near": None
+})
+
+LintMatch = namedtuple("LintMatch", MATCH_DICT.keys())
```

Comment:
You can provide default arguments by assigning a tuple to `LintMatch.__new__.__defaults__`, i.e. 

```py
LintMatch = namedtuple("LintMatch", MATCH_DICT.keys())
LintMatch.__new__.__defaults__ = tuple(MATCH_DICT.values())
```

Reply:
Thanks. Didn't know that one!

Reply:
@FichteFoll  `.values()` ? Shouldn't it be keys?